core/src/main/resources/hudson/diagnosis/ReverseProxySetupMonitor/resources.js

```diff
@@ -0,0 +1,35 @@
+(() => {
+	var redirectForm = document.getElementById('redirect-error');
+	if (!redirectForm) {
+		console.warn('This script expects to have an element with id="redirect-error" to be working.');
```

Comment:
Can we return here to avoid one extra level of nesting by skipping the else clause?